ServiceNow page fixes (#4485)
chef/automate: components/docs-chef-io/content/automate/servicenow_integration_install.md

[\[edit on GitHub\]](https://github.com/chef/automate/blob/master/components/docs-chef-io/content/automate/servicenow_integration_install.md)

This guide helps you set up a ServiceNow instance that creates incidents from Chef Automate failure notifications on failed Chef Infra Client runs and Compliance scans.

## Chef Automate ServiceNow Integration

The integration between a Chef Automate server and a ServiceNow instance requires the following:

* Chef Automate
* Chef Automate Incident Creation Application in ServiceNow

[Chef Automate](https://www.chef.io/automate/) provides a full suite of enterprise capabilities for workflow, node visibility, and compliance.
The Chef Automate Incident Creation Application is a ServiceNow certified scoped application available from the ServiceNow store.

> The ServiceNow Chef Automate Incident Creation Application creates or updates incidents based on HTTPS JSON notifications sent from your Chef Automate installation.

Reviewer: The app gets called by three different names in this section:

- Chef Automate ServiceNow Integration
- Chef Automate Incident Creation Application
- ServiceNow Chef Automate Incident Creation Application

Settle on one way of naming it and enforce that wherever it is used in this doc.

Author: You bring up a great point. Let me add to the confusion in wording and note that we have two apps in the ServiceNow Store named:
- [Chef Automate Integration App](https://store.servicenow.com/sn_appstore_store.do#!/store/application/02bf0043dba2d30070cd5e25ca9619a3/1.0.24?referer=%2Fstore%2Fsearch%3Flistingtype%3Dallintegrations%25253Bancillary_app%25253Bcertified_apps%25253Bcontent%25253Bindustry_solution%25253Boem%25253Butility%26q%3Dautomate&sl=sh)
- [Chef Automate Incident Creation](https://store.servicenow.com/sn_appstore_store.do#!/store/application/02bf0043dba2d30070cd5e25ca9619a3/1.0.24?referer=%2Fstore%2Fsearch%3Flistingtype%3Dallintegrations%25253Bancillary_app%25253Bcertified_apps%25253Bcontent%25253Bindustry_solution%25253Boem%25253Butility%26q%3Dautomate&sl=sh)

Author: I confirmed that this documentation is specifically for the Chef Automate Incident Creation application.

* Chef Automate
* Chef Automate Incident Creation Application in ServiceNow

> The Chef Automate Incident Creation application is a ServiceNow certified scoped application available from the ServiceNow store. [Chef Automate](https://www.chef.io/automate/) provides a full suite of enterprise capabilities for workflow, node visibility, and compliance.

Reviewer: ```suggestion
[Chef Automate](https://www.chef.io/automate/) provides a full suite of enterprise capabilities for workflow, node visibility, and compliance.
The Chef Automate Incident Creation application is a ServiceNow certified scoped application available from the ServiceNow store.
```

Reviewer: Move from the general to the specific & also stay consistent with the bullet points.

> Once configured, the Chef Automate server sends HTTPS JSON notifications to the Chef Automate Incident Creation Application in a ServiceNow instance to create or update incidents.

Reviewer: ```suggestion
That sends HTTPS JSON notifications from your Chef Automate installation to the ServiceNow Chef Automate Incident Creation Application to create or update incidents.
```
Add to the preceding paragraph & edit how they come together.

Author: @kagarmoe Do you want to add or replace line 27? Also what does "that" refer to in your suggested sentence?

Reviewer: I'm leaving that to you with `& edit how they come together`.
